app/src/main/java/com/example/xpvehicles/adapters/RentingVehiclesAdapter.java

```diff
@@ -136,23 +134,28 @@ public void setValues(RentVehicle rentvehicle) throws ParseException{
         }
 
         private void setStatusColor(String status) {
+            final String STATUS_PENDING_APPROVAL = "pending approval";
+            final String STATUS_APPROVED = "approved";
+            final String STATUS_PENDING_DENIED = "denied";
```

Comment:
You can use enum to represent these states

Reply:
I created this class, but am unsure how to access the values I saved

package com.example.xpvehicles.Miscellaneous;

import androidx.annotation.NonNull;

public enum RentingStatus {
    PENDING_APPROVAL ("pending status"),
    READY ("approved"),
    DENIED ("denied"),
    ;

    private String theStatus;

    RentingStatus(String s) {
        theStatus =  s;
    }

    @NonNull
    @Override
    public String toString() {
        return theStatus;
    }
}